docs/pages/versions/v49.0.0/config/app.mdx

```diff
@@ -7,7 +7,7 @@ maxHeadingDepth: 5
 {/* Hi! If you found an issue within the description of the manifest properties, please create a GitHub issue. */}
 
 import AppConfigSchemaPropertiesTable from '~/components/plugins/AppConfigSchemaPropertiesTable';
-import schema from '~/public/static/schemas/unversioned/app-config-schema.json';
+import schema from '~/public/static/schemas/v49.0.0/app-config-schema.json';
```

Comment:
Happy to separate out this fix from the rest of this PR if reviewers prefer

Reply:
I think it's fine to have these fixes in this PR.